packages/node-cli/src/services/TranslationKeys.js

```diff
@@ -255,6 +255,8 @@ module.exports = {
 		HELP: 'COMMAND_OPTIONS_HELP',
 		INTERACTIVE_HELP: 'COMMAND_OPTIONS_INTERACTIVE_HELP',
 		IS_MANDATORY: 'COMMAND_OPTIONS_IS_MANDATORY',
+		IS_MANDATORY_SELECT_MODE: 'COMMAND_OPTIONS_IS_MANDATORY_SELECT_MODE',
+		IS_NOT_ALLOWED_SELECT_MODE: 'COMMAND_OPTIONS_IS_NOT_ALLOWED_SELECT_MODE',
```

Comment:
This are general command options, this specific one for `account:setup:ci` should be part of `COMMAND_SETUPACCOUNTCI`